python/tank/util/shotgun/connection.py

```diff
@@ -411,9 +419,9 @@ def create_sg_connection(user="default"):
         # Credentials were passed in, so let's run the legacy authentication
         # mechanism for script user.
         api_handle = shotgun_api3.Shotgun(
-            config_data["host"],
-            script_name=config_data["api_script"],
-            api_key=config_data["api_key"],
+            os.path.expandvars(config_data["host"]),
```

Comment:
`__get_sg_config_data_with_script_user` already does the substitution, so these shouldn't be required, correct?

Reply:
You're right. Fixed!